frontend/micro-ui/web/micro-ui-internals/packages/components-core/src/hoc/stories/Fields.Stories.js

```diff
@@ -0,0 +1,279 @@
+import React from "react";
+import Fields from "../Fields";
+
+export default {
+  title: "Atom-Groups/Fields",
+  component: Fields,
+  argTypes: {
+    type: {
+      control: {
+        type: "select",
+        options: [
+          "date",
+          "text",
+          "number",
+          "password",
+          "time",
+          "amount",
+          "textarea",
+          "paragraph",
+          "mobileNumber",
+          "custom",
+          "checkbox",
+          "multiupload",
+          "select",
+          "radio",
+          "dropdown",
+          "radioordropdown",
+          "component",
+          "documentUpload",
+          "form",
+          "locationdropdown",
+          "apidropdown",
+          "multiselectdropdown",
+        ],
+      },
+    },
+  },
+};
+
+const Template = (args) => <Fields type={args?.type} {...args} />;
+
+export const Playground = Template.bind({});
+Playground.args = {
+  type: args?.type,
+  populators: {
+    name: "salutation",
+    error: "Required",
+    validation: {
+      pattern: {},
+      maxlength: 5,
+    },
+  },
+  isMandatory: false,
+  disable: false,
+  component: undefined,
+  config: {
+    inline: true,
+    label: "Salutation",
+    isMandatory: false,
+    type: "text",
+    disable: false,
+    populators: {
+      name: "salutation",
+      error: "Required",
+      validation: {
+        pattern: {},
+        maxlength: 5,
+      },
+    },
+  },
+  sectionFormCategory: undefined,
+  formData: {},
+  selectedFormCategory: undefined,
+  props: {
+    label: "Submit Bar",
+    config: [
+      {
```

Comment:
why this config is required
some syntax error in this component
revisit @nabeelmd-eGov

Reply:
@jagankumar-egov reduces the config. some config is required to render field as they are mandatory one.